x-pack/plugins/ml/public/application/datavisualizer/file_based/components/results_links/results_links.tsx

```diff
@@ -148,23 +169,6 @@ export const ResultsLinks: FC<Props> = ({
     }
   }
 
-  function openInDiscover(e: React.MouseEvent<HTMLButtonElement>) {
-    e.preventDefault();
-    navigateToUrl(discoverLink);
-  }
-
-  function openIndexManagement(e: React.MouseEvent<HTMLButtonElement>) {
-    e.preventDefault();
-    navigateToApp('management', { path: '/data/index_management/indices' });
-  }
-
-  function openIndexPatternManagement(e: React.MouseEvent<HTMLButtonElement>) {
-    e.preventDefault();
-    navigateToApp('management', {
-      path: `/kibana/indexPatterns${createIndexPattern ? `/patterns/${indexPatternId}` : ''}`,
-    });
-  }
-
   return (
     <EuiFlexGroup gutterSize="l">
       {createIndexPattern && discoverLink && (
```

Comment:
do you need to add a check for all the other links as well?

Reply:
Updated here https://github.com/elastic/kibana/pull/80785/commits/0064254121231a7b0d852d86dbda090d4e47b78d